### ISSUE TITLE
-uncoverable_file is not considered by the cover utility

### ISSUE THREAD

**Issue by emiliaboyadjieva** [pjcj/Devel--Cover#218]
I have run the cover utility with -uncoverable_file and uncoverable points were not considered as such. I have seen that there is a # TODO - read in and merge $self->uncoverable; but this option is documented as implmented

**Comment by pjcj** [comment]
Yes, it's quite likely there are errors in this area.  I wrote the code, but it was never fully finished so I didn't document it.  Then people started finding the code and using it so I thought I might as well document what was there, but maybe I documented what should have been there ;)

Initially I had expected the uncoverable file to have been the most popular way annotating uncoverable code, but most people seemed happier adding comments in the source, so I let other things take priority over the uncoverable files.  Is this something you would use if it worked well?

Thanks very much for noticing and reporting this.
